raddb/mods-available/ldap

```diff
@@ -135,231 +191,287 @@ ldap {
+			#
+			#  mech:: SASL mechanism.
+			#
 #			mech = 'PLAIN'
 
-			# SASL authorisation identity to proxy.
+			#
+			#  proxy:: SASL authorisation identity to proxy.
+			#
 #			proxy = &User-Name
 
-			# SASL realm. Used for kerberos.
+			#
+			#  realm:: SASL realm. Used for kerberos.
+			#
 #			realm = 'example.org'
 		}
 
-		#  Search scope, may be 'base', 'one', sub' or 'children'
+		#
+		#  scope:: Search scope, may be `base`, `one`, `sub' or `children`.
+		#
 #		scope = 'sub'
 
-		#  Server side result sorting
+		#
+		#  sort_by:: Server side result sorting.
+		#
+		#  [NOTE]
+		#  ====
 		#
 		#  A list of space delimited attributes to order the result set by.
 		#
-		#  If the filter matches multiple objects only the first
+		#  * If the filter matches multiple objects only the first
 		#  result will be processed.
 		#
-		#  If the attribute name is prefixed with a hyphen '-' the
+		#  * If the attribute name is prefixed with a hyphen '-' the
 		#  sorting order will be reversed for that attribute.
 		#
-		#  If sort_by is set, and the server does not support sorting
+		#  * If sort_by is set, and the server does not support sorting
 		#  the search will fail.
 		#
-		#  If a search returns multiple user objects and sort_by is not
+		#  * If a search returns multiple user objects and `sort_by` is not
 		#  set, the search will fail.
+		#  ====
+		#
 #		sort_by = '-uid'
 
-		#  If this is undefined, anyone is authorised.
-		#  If it is defined, the contents of this attribute
-		#  determine whether or not the user is authorised
+		#
+		#  access_attribute:: If this is undefined, anyone is authorised.
+		#
+		#  If it is defined, the contents of this attribute determine whether
+		#  or not the user is authorised.
+		#
 #		access_attribute = 'dialupAccess'
 
-		#  Control whether the presence of 'access_attribute'
+		#
+		#  access_positive:: Control whether the presence of `access_attribute`
 		#  allows access, or denys access.
 		#
-		#  If 'yes', and the access_attribute is present, or
-		#  'no' and the access_attribute is absent then access
-		#  will be allowed.
+		#  [NOTE]
+		#  ====
+		#  If `yes`, and the `access_attribute` is present, or `no` and the
+		#  `access_attribute` is absent then access  will be allowed.
 		#
-		#  If 'yes', and the access_attribute is absent, or
-		#  'no' and the access_attribute is present, then
-		#  access will not be allowed.
+		#  If `yes`, and the `access_attribute` is absent, or `no` and the
+		#  `access_attribute` is present, then access will not be allowed.
 		#
-		#  If the value of the retrieved access_attribute is
-		#  'false', it will negate the result.
+		#  If the value of the retrieved `access_attribute` is `false`, it will
+		#  negate the result.
 		#
-		#  e.g.
+		#  e.g:
+		#
+		#  ```
 		#    access_positive = yes
 		#    access_attribute = userAccessAllowed
+		#  ```
 		#
 		#  With an LDAP object containing:
+		#
+		#  ```
 		#    userAccessAllowed: false
+		#  ```
```

Comment:
Better to use consistent formatting, i.e. a codeblock maybe even mark it up as ldiff